app/src/main/res/values/array.xml

```diff
@@ -7,4 +7,13 @@
         <item>100</item>
         <item>500</item>
     </string-array>
+    <string-array name="tabs">
+        <item>@string/menu_dices</item>
+        <item>@string/menu_random_list</item>
+        <item>@string/menu_rot13</item>
+        <item>@string/menu_alphabet</item>
+        <item>@string/menu_number_converter</item>
+        <item>@string/menu_timer</item>
+        <item>@string/menu_buzzers</item>
+    </string-array>
```

Comment:
These strings may change, e.g. a translation is added. Can you add non-translatable strings by appending `_value`, e.g. `menu_dices_value`?
Then add a second array `tabs_value` and set it as `entryValues` in the `ListPreference`.

Reply:
Sorry got to be honest im a little bit confused here...
So I should add to `strings.xml` non-translate-able values for each tab title?
And then use that in this `tabs` array?

Sorry little confused here

Reply:
basically one string for the prefs key and one for the display hahaha got it